controllers/err-controllers.js

```diff
@@ -0,0 +1,16 @@
+
+
+// const handle400Erros = (req, res, next) => {
+//     res.status(404).send({ msg: "incorrect endpoint" });
+//   };
+  
+
+const handle500Errors = (error, req, res, next) => {
+    if (error.status === 500) {

```

Comment:
This will trigger for ALL 500 errors and won't help us pin down the actual error. You can find codes in the error that comes back to help you create a custom error